os/include/tinyara/mpu.h

```diff
@@ -96,4 +96,6 @@ enum MPU_REG_NUM {
 
 void mpu_get_register_value(uint32_t *regs, uint32_t region, uintptr_t base, size_t size, uint8_t readonly, uint8_t execute);
 
+void up_mpu_set_register(uint32_t *mpu_regs);
+
```

Comment:
I think we need to include this file in all other files where we use this API. Pls check.

Reply:
done